packages/apps/tools/src/pages/faucet/new/index.tsx

```diff
@@ -45,6 +44,8 @@ import { createPrincipal } from '@/services/faucet/create-principal';
 import { fundCreateNewAccount } from '@/services/faucet/fund-create-new';
 import { validatePublicKey } from '@/services/utils/utils';
 import { zodResolver } from '@hookform/resolvers/zod';
+import type { TPredicate } from '@kadena/types';
+import { Predicate } from '@kadena/types';
```

Comment:
I think the type library should only contain Types? So we might need to place this somewhere else. @alber70g what do you think?